src/modules/transaction/components/Multisignature/Multisignature.js

```diff
@@ -114,7 +195,27 @@ const Multisignature = ({
       <Illustration name={getIllustration(status.code, 'signMultisignature')} />
       <h6 className="result-box-header">{title}</h6>
       <p className="transaction-status body-message">{message}</p>
-
+      {nextAccountToSign && (
+        <div className={styles.requiredAccountSection}>
+          <WarningNotification
+            isVisible
+            message={
+              <span>
+                {t('A required signatory account')}{' '}
+                <b>
+                  ({nextAccountToSign?.metadata?.name} -
+                  {truncateAddress(nextAccountToSign?.metadata?.address)})
```

Comment:
Some space should be added before the account address to maintain consistency with the account name. For example, `genesis1 -lsk5px...bz76y` should be `genesis1 - lsk5px...bz76y`

Reply:
Fixed